app/views/steps/miam/no_certification_kickout/show.en.html.erb

```diff
@@ -0,0 +1,21 @@
+<% title 'You need a MIAM certificate' %>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%= step_header %>
+
+    <!-- This is a temporal placeholder -->
```

Comment:
You mean 'temporary'?  It that why the copy isn't in the translations?

Reply:
Yep, temporary 😄

The fact it is not in translations is in effect because we don't know yet the final copy but also because for some static copy-heavy steps, without form objects, fields, etc. for this project we are going with translated files instead i.e. `show.en.html.erb` for english, etc. as that makes it easier to update/maintain than having a lot of keys in i18n or a huge key with embedded html.

Reply:
That's fine.  Can you change 'temporal' to 'temporary', please?